torch/utils/serialization/_config.py

```diff
@@ -0,0 +1,26 @@
+import sys
+from typing import Optional
+
+from torch.serialization import LoadEndianness
+from torch.utils._config_module import install_config_module
+
+
+IS_WINDOWS = sys.platform == "win32"
+
+if not IS_WINDOWS:
+    from mmap import MAP_PRIVATE, MAP_SHARED
+else:
+    MAP_SHARED, MAP_PRIVATE = None, None  # type: ignore[assignment]
+
+
+class load:
+    mmap: bool = False
+    endianness: Optional[LoadEndianness] = None
```

Comment:
Weird question - Can we do

``
Optional["LoadEndianness"]
```

and above

```
if TYPECHECKING:
  from torch.serialization import LoadEndianness
```

Reply:
This seems to work :)